format.sh

```diff
@@ -25,8 +34,11 @@ find . -not -path "./third_party/**" \
   | xargs clang-format --verbose -style=google -i
 
 # Run buildifier (Bazel file formatter).
-BUILDIFIER="buildifier-$PLATFORM"
-wget "https://github.com/bazelbuild/buildtools/releases/download/v7.3.1/$BUILDIFIER"
-mv $BUILDIFIER buildifier
-chmod +x buildifier
+if [ ! -e buildifier ]
```

Comment:
Does this logic here need to be wrapped into a conditional?

Reply:
It does not need to be, but if you as a developer run format.sh locally more than once, while making a sequence of changes over time, it is nice not to download and install buildifier every single time, only the first time, which is what this conditional is doing.

Reply:
That makes sense, sounds good.